style/style.css

```diff
@@ -383,7 +383,7 @@ body > footer {
 		top: 0;
 		bottom: 0;
 		z-index: -1;
-		width: 50vw;
+		width: 49.6vw;
```

Comment:
Yeah, that is kinda hacky. I'd rather figure out *why* the scrollbar is appearing first…